<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DustMonSysTClass</class>
 <widget class="QMainWindow" name="DustMonSysTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DustMonSysT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>扬尘检测</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <widget class="QPushButton" name="pbHistory">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>100</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>历史</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pbNow">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>40</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>实时</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>150</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>200</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Test</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>PM2.5</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>PM10</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>湿度</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>温度</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLCDNumber" name="lcdPm25"/>
            </item>
            <item>
             <widget class="QLCDNumber" name="lcdPm10"/>
            </item>
            <item>
             <widget class="QLCDNumber" name="lcdTem"/>
            </item>
            <item>
             <widget class="QLCDNumber" name="lcdHum"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>μg/m3</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>μg/m3</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>℃</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>%</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QWidget" name="widget_2" native="true">
            <widget class="QTableWidget" name="tableWidget">
             <property name="geometry">
              <rect>
               <x>100</x>
               <y>80</y>
               <width>461</width>
               <height>271</height>
              </rect>
             </property>
             <row>
              <property name="text">
               <string>新建行</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>Pm25</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Pm10</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>温度</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>湿度</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>时间</string>
              </property>
             </column>
             <item row="0" column="0">
              <property name="text">
               <string>pm10</string>
              </property>
             </item>
             <item row="0" column="2">
              <property name="text">
               <string>pm25</string>
              </property>
             </item>
             <item row="0" column="3">
              <property name="text">
               <string>52</string>
              </property>
             </item>
             <item row="0" column="4">
              <property name="text">
               <string>25</string>
              </property>
             </item>
            </widget>
            <widget class="QPushButton" name="pushButton_2">
             <property name="geometry">
              <rect>
               <x>150</x>
               <y>30</y>
               <width>93</width>
               <height>28</height>
              </rect>
             </property>
             <property name="text">
              <string>刷新</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DustMonSysT.qrc"/>
 </resources>
 <connections/>
</ui>
